ksa_compliance/ksa_compliance/doctype/sales_invoice_additional_fields/sales_invoice_additional_fields.py

```diff
@@ -51,7 +51,7 @@ class SalesInvoiceAdditionalFields(Document):
         buyer_country_code: DF.Data | None
         buyer_district: DF.Data | None
         buyer_postal_code: DF.Data | None
-        buyer_provincestate: DF.Data | None
+        buyer_province_state: DF.Data | None
```

Comment:
Are we setting this anywhere? We currently set buyer details in sales_invoice.py, so we should either fix the reference there or add it if it doesn't exist.